packages/storybook/stories/global-navigation.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import { GlobalNavigation, GlobalNavigationItem } from '@equisoft/design-elements-react';
+import { IconButtonProps } from '@equisoft/design-elements-react/dist/components/buttons/icon-button';
```

Comment:
Il est peut-être pas exporté, alors faudrait faire ça en plus.

```suggestion
import { GlobalNavigation, GlobalNavigationItem, IconButtonProps } from '@equisoft/design-elements-react';
```